### ISSUE TITLE
IconRequest : Network Diagnostics

### ISSUE THREAD

**Issue by TheQuantum** [tinalatif/flat.icns#161]
This isn’t so much a request for an icon as a request for feedback.  I’m new to the whole graphic design thing but I made an icon for Network Diagnostics (I’ve been having a lot of network problems lately so it’s open a lot) and I wanted feedback on it before submitting a pull request.

The Original:
![Network Diagnostics](https://cloud.githubusercontent.com/assets/10932585/7731019/f469171e-ff1d-11e4-8f81-558388a4d8da.png)
My Version:
![network diagnostics](https://cloud.githubusercontent.com/assets/10932585/7731038/0b474514-ff1e-11e4-8bcf-a0871e678481.png)

**Comment by wjramos** [comment]
I like the tools a lot, I think the globe seems a bit overshadowed. I feel like it should be larger, but I'm not sure how to do that without covering up the tools more (maybe have it behind the tools?). I also think the details in the globe could be a bit cleaner, or removed altogether

**Comment by tinalatif** [comment]
I agree with almost all of of @wjramos's comments. Only thing I disagree with is having the globe behind the tools or removing the nwtwork details - I think it would take away from the primary "point" of the icon (without that, it just looks like any generic settings or config app).

I think the details should be cleaned up though. They're currently tapered and some of them end halfway across the globe, which makes it look kinda... spider web-y?  I'm not sure it'll look great, but what I would try is to get rid of the tapering and just have constant-width lines running across the globe that cut off at the edges of the globe (with nodes intact). And maybe make the network a bit brighter/whiter?

**Comment by TheQuantum** [comment]
A new version.  The globe is ever so slightly bigger but increasing it much more made it look too bulbous to me (it might be already, what do you think?).  
![network diagnostics](https://cloud.githubusercontent.com/assets/10932585/8576336/8c6cfcca-25a1-11e5-8af9-be43bc32046e.png)

**Comment by wjramos** [comment]
I actually think this is perfect - It looks so much better with the brighter, more solid network lines, and I think the globe is the perfect size
